Ginger/GingerCoreNET/ActionsLib/ActFileOperations.cs

```diff
@@ -167,6 +168,21 @@ public override void Execute()
                         System.IO.File.Delete(file);
                     }
                     break;
+
+                case eFileoperations.DeleteDirectory:
+                    if (!System.IO.Directory.Exists(calculatedSourceFilePath))
+                    {
+                        base.Status = Amdocs.Ginger.CoreNET.Execution.eRunStatus.Passed;
```

Comment:
Action should be failed in case directory is not present

Reply:
Modified the code now for handling failed scenarios.